src/foam/u2/view/LoginView.js

```diff
@@ -432,6 +432,7 @@ foam.CLASS({
     },
     function onKeyPressed(e) {
       e.preventDefault();
+      if ( this.document.hasFocus() ) return;
```

Comment:
But we add the listener like this:
this.document.addEventListener('keyup', this.onKeyPressed);
Why do we add it to the document in the first place instead of just the textfield?

Reply:
i guess its to login on enter even without focus on the textfield, but not sure if its really necessary..  should i just remove this?

Reply:
Yes, and set autofocus on the text field.